bootstrap/commandline/commandline.go

```diff
@@ -25,8 +25,8 @@ const (
 	DefaultConfigFile     = "configuration.toml"
 )
 
-// CommonFlags is an interface that defines AP for the common command-line flags used by most EdgeX services
-type CommonFlags interface {
+// Flags is an interface that defines AP for the common command-line flags used by most EdgeX services
```

Comment:
I don't agree with this renaming.  `CommonFlags` is explicit that encapsulates the common flags and `Flags` doesn't . From a usage POV, `Flags` just looks to be an alternative implementation and doesn't imply that it actually add all the common flags.

Reply:
Also, per @tsconn23, bootstrap/commandline/commandline.go is name stuttering. I googled this and didn't find anything on filename stuttering. I did find package/struct stuttering guidelines, which isn't an issue here. ;-)

Reply:
> Also, per @tsconn23, bootstrap/commandline/commandline.go is name stuttering. I googled this and didn't find anything on filename stuttering. I did find package/struct stuttering guidelines, which isn't an issue here. ;-)

I have previously discussed this with @tsconn23 and convinced him that filename stutter isn't stutter we're concerned about.  See https://github.com/edgexfoundry/go-mod-bootstrap/blob/master/bootstrap/bootstrap.go for a representative existing example.